### PR TITLE
An X symbol appears when a user clicks on a spot.

### PR DESCRIPTION
**Pull request by ericdrosado**

### CODE REVIEW

File: bundle.js [hunk 1]
```diff
@@ -18232,14 +18232,15 @@ var TicTacToe = function() {
 TicTacToe.prototype.main = function() {
```
Comment (NicoleCarpenter) [inline bundle.js]: Should this file be checked in? ¯\_(ツ)_/¯

File: spec/UISpec.js [hunk 2]
```diff
@@ -9,15 +9,16 @@ describe("UI", function() {
...
-    it("display the turn if it's W's turn", function() {
-      ui.displayTurn("W");
-      expect($("#turn-label")).toHaveText("W wins!!");
+    it("can display a marker when a spot is clicked", function() {
+      ui.listenForSpotClick(function(e){$(e.target).html("X")});
+      $("#0").trigger("click");
+      expect($("#0")).toHaveText("X");
     });
-
+
```
Comment (NicoleCarpenter) [inline spec/UISpec.js]: 👾

File: src/TicTacToe.js [hunk 3]
```diff
@@ -9,14 +9,15 @@ var TicTacToe = function() {
 TicTacToe.prototype.main = function() {
 
   this.ui = new UI();
-  ui.listenForSpotClick(TicTacToe.prototype.spotClicked);
+  this.ui.listenForSpotClick(TicTacToe.prototype.spotClicked);
```
Comment (NicoleCarpenter) [inline src/TicTacToe.js]: Good naming overall 👍

File: src/games/PlayerVsPlayerGame.js [hunk 4]
```diff
@@ -6,4 +6,7 @@ PlayerVsPlayerGame.prototype.play = function() {
   this.ui.displayTurn("X");
 }
 
+
+
+
```
Comment (NicoleCarpenter) [inline src/games/PlayerVsPlayerGame.js]: 👾

File: test_bundle.js [hunk 5]
```diff
@@ -11115,9 +11115,9 @@ describe("PlayerVsPlayerGame", function() {
       game.play();
```
Comment (NicoleCarpenter) [inline test_bundle.js]: Probably also should not check this in.

File: bundle.js (hunk 1 above)
Comment (tommcgee) [inline bundle.js]: ^ Probably not!

File: src/ui.js [hunk 6]
```diff
@@ -1,20 +1,31 @@
 var $ = require('jquery');
 
 function UI() {
+  var playerMarker = ""
 }
 
 UI.prototype.displayTurn = function(marker) {
   $("#turn-label").html(marker + "'s turn!");
+  playerMarker = marker;
```
Comment (tommcgee) [inline src/ui.js]: I think you should probably be using `this.playerMarker` here

File: src/ui.js [hunk 7]
```diff
@@ -1,20 +1,31 @@
 var $ = require('jquery');
 
 function UI() {
+  var playerMarker = ""
```
Comment (tommcgee) [inline src/ui.js]: Like below, I think this should be `this.playerMarker`

File: src/ui.js [hunk 8]
```diff
@@ -1,20 +1,31 @@
...
+
+UI.prototype.listenForSpotClick = function(spotClicked) {
+  this.callOnElementClick(".spot", spotClicked);
 }
 
 UI.prototype.callOnElementClick = function(element, callback) {
   $(element).on("click", function(e) {
-    callback(e);
+    var id = callback(e);
+    UI.prototype.displayMarker(id, playerMarker);
```
Comment (tommcgee) [inline src/ui.js]: `this`

File: test_bundle.js (hunk 5 above)
Comment (tommcgee) [inline test_bundle.js]: 👍

File: src/ui.js (hunk 7 above)
Comment (ericdrosado) [inline src/ui.js]: When I add "this" I get an error ```Error: Parsing file /Users/ericrosado/Documents/8thLightApps/JSTTT/tttaas-js/src/ui.js: Unexpected token (4:6)```

**Pull request merged**